db/migrate/20201118142202_add_uuid_columns_to_lighthouse_samples.rb

```diff
@@ -0,0 +1,8 @@
+class AddUuidColumnsToLighthouseSamples < ActiveRecord::Migration[6.0]
+  def change
+    change_table :lighthouse_sample, bulk: true do |t|
+      t.string :lh_sample_uuid, limit: 36, null: true, comment: 'Sample uuid created in crawler', after: :filtered_positive_timestamp
+      t.string :lh_source_plate_uuid, limit: 36, null: true, comment: 'Source plate uuid created in crawler', after: :lh_sample_uuid
+    end
```

Comment:
Sorry for not knowing, what is ```after`` for?

Reply:
'after' positions the new column relative to an existing one, so add this column after filtered_positive_timestamp column

Reply:
by default it would dump them on the end after the create and update timestamps